toolkit/tools/imagegen/installutils/installutils.go

```diff
@@ -758,9 +763,16 @@ func addMachineID(installChroot *safechroot.Chroot) (err error) {
 
 	ReportAction("Configuring machine id")
 
-	err = installChroot.UnsafeRun(func() error {
-		return file.Create(machineIDFile, machineIDFilePerms)
-	})
+	exists, err := file.PathExists(filepath.Join(installChroot.RootDir(), machineIDFile))
+	if err != nil {
+		err = fmt.Errorf("unable to check if machine-id exists. Error:\n%w", err)
```

Comment:
nit:
to keep it consistent with error handling in utlity, could you try
`err = fmt.Errorf("failed to check if machine-id exists:\n%w", err)`

Reply:
I think we use `Error:\n%w` about 50/50 in other code, and this file is all "Error: ". I'm fine with simplifying but we should do that in one pass I think.

Reply:
Yes, cleaning up the error logging in toolkit so it's uniform

Reply:
```suggestion
		err = fmt.Errorf("failed to check if machine-id exists:\n%w", err)
```